Storage/SQL/CommandSyncTable.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CommandSyncTable.swift
+//  Client
+//
+//  Created by Emily Toop on 6/29/15.
+//  Copyright (c) 2015 Mozilla. All rights reserved.
+//
+
+import Shared
+
+class CommandSyncTable<T>: GenericTable<SyncCommand> {
```

Comment:
I think you might be splitting the difference between generic and specific here.

Generically, a command-send is a JSON blob: `{command: "foo", args: ["some", "args"]}` and a client GUID: what and who.

Specifically, a displayURI command is a particular command name and a particular list of args: URL, clientID, title.

This table seems to be neither one nor the other. Might be worth deciding if you want to just go full generic and store JSON as the 'value' column, or have command and args, or make this specific.